string[5] foo
